frontend/src/components/candidateDropdown.js

```diff
@@ -60,24 +62,26 @@ class CandidateDropdown extends Component<Props, State> {
       this.setState({ selectedOption: newSelectedOption })
     }
   }
+
   render() {
-    const { isClearable, isSearchable, selectedOption } = this.state
+    const { selectedOption } = this.state
     const { candidates } = this.props
     var candidate_names = []
     for (var i = 0; i < candidates.length; i++) {
       let curr_cand = this.props.candidates[i]
       let curr_name_label = { value: curr_cand.name.toString(), label: curr_cand.name.toString() }
       candidate_names.push(curr_name_label)
     }
+
     return (
-      <Fragment>
+      <>
```

Comment:
nit: do we need a fragment if it only has one component?

Reply:
Nope - good catch! Will remove.